test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -83,6 +83,8 @@ build_branch()
         . ./test/evergreen/find_cmake.sh
         config=""
         config+="-DENABLE_SNAPPY=1 "
+        # Need to disable configs since WT-9455
+        config+="-DENABLE_LZ4=0 -DENABLE_ZLIB=0 -DENABLE_ZSTD=0 "
```

Comment:
Is this line added to bring the config of old branches in line with the new config from WT-9455? If so it'd be good to add in the comment.

Reply:
This line is to turn off new defaults since WT-9455 changed defaults that don't align. Will update.